site/en/install/pip.md

```diff
@@ -37,6 +44,26 @@ step-by-step instructions.
    python3 -c "import tensorflow as tf; print(tf.config.list_physical_devices('GPU'))"
    ```
 
+* {Windows WSL2}
+
+   Note: TensorFlow is supported on WSL2 on Windows 10 19044 or higher with GPU
+   access is now available. This corresponds to Windows 10 version
```

Comment:
Note: TensorFlow is supported on WSL2 on Windows 10 19044 or higher with GPU
   access available.

Reply:
Done.